src/features/elementfeature.jl

```diff
@@ -91,8 +105,20 @@ function (ed::OneHotOneCold)(
     end
 end
 
+"""
+    output_shape(efd::ElementFeatureDescriptor)
+
+Get the output-shape for an ElementFeatureDescriptor object using the logic assoicated with its
+Codec.
+"""
 output_shape(efd::ElementFeatureDescriptor) = output_shape(efd, efd.encoder_decoder)
 
+"""
+    output_shape(efd::ElementFeatureDescriptor, ed::OneHotOneCold)
```

Comment:
I'm not sure this actually needs to be a separate docstring?

Reply:
fair enough.